src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -30,6 +52,15 @@
 
   </Commands>
 
+  <!-- Short cut keybindings: see https://docs.microsoft.com/en-us/visualstudio/extensibility/binding-keyboard-shortcuts-to-menu-items?view=vs-2019 -->
+  <KeyBindings>
+    <KeyBinding guid="guidIssueVizPackageCmdSet" id="GotoNextLocationCommandId"
```

Comment:
It was hard to find free keys... ctrl+alt+arrows are used for resharper commands, ctrl+shift+arrows are used by VS.

I thought we could bind it to the num pad, with ctrl+4/6 for locations, and ctrl+5/8 for flows. 

In either case, the commands are in the command well, and will appear as "Tools.SonarLintGoToPreviousLocation" and "Tools.SonarLintGoToNextLocation".

Reply:
As discussed offline, lots of laptops/keyboards don't have numeric keypads.
There might be a suitable key chord combination we could use.